core/new-gui/src/app/dashboard/component/feature-container/saved-workflow-section/saved-workflow-section.component.ts

```diff
@@ -847,8 +808,19 @@ export class SavedWorkflowSectionComponent implements OnInit, OnChanges {
         if (this.userService.isLogin()) {
           this.refreshDashboardWorkflowEntries();
           this.refreshUserProjects();
+          this.zip = new JSZip();
+          this.downloadListWorkflow = [];
         } else {
+          for (let wid of this.downloadListWorkflow) {
+            console.log(wid);
+            const checkbox = document.getElementById(wid.toString()) as HTMLInputElement | null;
```

Comment:
Maybe this logic can be moved into `clearDashboardWorkflowEntries`?

Reply:
Done.